### PR TITLE
chore: add docker db setup 

### PR DESCRIPTION
**Pull request by zo-chu**
- adds a docker compose file with a Postgres db, so contributors can run tests without installing the db
- update Readme

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    image: postgres:9-alpine
+    restart: always
+    environment:
+      - POSTGRES_USER=wallet_user
+      - POSTGRES_PASSWORD=secret
+    ports: 
+      - 5432:5432
+    volumes: 
+      - postgres_volume:/var/lib/postgresql
+      - ./init-db.sql:/docker-entrypoint-initdb.d/init-db.sql
```
Comment (dadiorchen) [inline docker-compose.yml:14]: I think this brings big convenience, thanks.
@mzyau about the init SQL, do we still are using it for set up DB? or just need to use db-migrate?

**Review by dadiorchen — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (mzyau) [reply docker-compose.yml:14]: Yup, we no longer need to seed the db with a sql file since we can use migrations now.

Comment (zo-chu) [reply docker-compose.yml:14]: Cool! I was thinking to set up docker to run migration as well, but probably can do that later as a separate thing

Comment (zo-chu) [reply docker-compose.yml]: @dadiorchen I've removed the init script, so I think this could be merged, unless anything else needs changing. Thanks

**Pull request merged**